cluster-up/cluster/kind-k8s-sriov-1.17.0/README.md

```diff
@@ -41,4 +42,33 @@ export KIND_VERSION="0.9.0"
 export KUBECTL_PATH="/usr/bin/kubectl"
 ```
 This allows users to test or use custom images / different kind versions before making them official.
-See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
\ No newline at end of file
+See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
+
+## Running multi sriov clusters locally
+Kubevirtci sriov provider supports running two clusters side by side with few known limitations.
+
+General considerations:
+
+- A sriov PF must be available for each cluster.
+In order to achieve that, there are two options:
+1. Assign just one PF for each worker node of each cluster by using `export PF_COUNT_PER_NODE=1` (this is the default value).
+2. Optional method: `export PF_BLACKLIST=<PF names>` the non used PFs, in order to prevent them from being allocated to the current cluster.
+The user can list the PFs that should not be allocated to the current cluster, keeping in mind
+that at least one (or 2 in case of migration), should not be listed, so they would be allocated for the current cluster.
+Note: another reason to blacklist a PF, is in case its has a defect or should be kept for other operations (for example sniffing).
+- The cluster names must be different.
+This can be achieved by setting `export CLUSTER_NAME=sriov2` on the 2nd cluster.
+The default `CLUSTER_NAME` is `sriov`.
+The 2nd cluster registry would be exposed at `localhost:5001` automatically, once the `CLUSTER_NAME`
+is set to a non default value.
+- Each cluster should be created on its own git clone folder, i.e
+`/root/project/kubevirtci1`
+`/root/project/kubevirtci2`
+In order to switch between them, change dir to that folder and set the env variables `KUBECONFIG` and `KUBEVIRT_PROVIDER`.
+- In case only one PF exists, for example if running on prow which will assign only one PF per job in its own DinD,
+Kubevirtci is agnostic and nothing needs to be done, since all conditions above are met.
+The number of clusters that can be run on the same time equals number of PFs / number of PFs per cluster,
```

Comment:
If this is true then we have to change this https://github.com/kubevirt/kubevirtci/pull/551/files#diff-fcaa29a18b4077ecaa46dca6e69b637432306c3d2f38d3def35ff59ecc33dbbfR7 so we can have random number of registries right ?

Reply:
Yes, but we now support only 2 at the same time, and its for local use only.
It doesn't affect CI, since CI runs in DinD, it can use the same port.
Its just a general readme note, i can add that we are limited to 2 locally.

Reply:
Updated